www/includes/easyparliament/templates/html/section/column.php

```diff
@@ -0,0 +1,431 @@
+<?php
```

Comment:
This file overlaps a lot with section.php, it seems - any chance it could use section.php instead?